index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Mappings for Verifiable Credentials</h3>
+
+        <p>
+The standard representation of a Verifiable Credential is defined in this specification (which uses the media type <code>application/vc+ld+json</code>).
+Other representations can be used, provided that a mapping is defined between that representation and the standard one.
+This mapping definition should state whether it is unidirectional or bidirectional.
```

Comment:
This sounds like an RFC SHOULD <-- instruction to specification author.

Reply:
@manu, I've accepted your changes and also accepted @TallTed 's suggestions which deal with the use of "should".  Can you please re-review and possibly approve?  Thank you.